hystrix/src/main/java/feign/hystrix/HystrixDelegatingContract.java

```diff
@@ -0,0 +1,37 @@
+package feign.hystrix;
+
+import static feign.Util.resolveLastTypeParameter;
+
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import java.util.List;
+
+import com.netflix.hystrix.HystrixCommand;
+
+import feign.Contract;
+import feign.MethodMetadata;
+
+public class HystrixDelegatingContract implements Contract {
```

Comment:
s/public/final unless you think folks should be extending this or using it directly